stylesheets/partials/overrides/_google-custom-search.scss

```diff
@@ -0,0 +1,148 @@
+/* Results */
+
+.gsc-resultsHeader,
+.gsc-location {
+    display: none !important;
+}
+
+.gsc-control-cse {
+    padding: 0 !important;
+}
+
+.gsc-result {
+    border: none !important;
+}
+
+.gsc-control-cse,
+.gsc-control-cse table tbody tr:nth-child(odd) td {
+    background: none !important;
+    border: none !important;
+}
+
+.gsc-control-cse table tbody tr:nth-child(odd) td {
+    font-size: 16px !important;
+    padding: 0;
+}
+
+.gsc-table-result,
+.gs-webResult .gs-snippet {
+    margin-bottom: 0 !important;
+}
+
+.gsc-table-result,
+.gsc-thumbnail-inside,
+.gsc-url-top {
+    padding: 0 !important;
+}
+
+.gsc-webResult.gsc-result {
+    @extend .card;
+    padding: 16px !important;
+    margin-bottom: 15px !important;
+}
+
+.gs-bidi-start-align.gs-snippet {
+    font-size: 16px !important;
+}
+
+/* Result Title */
+
+.gsc-result .gs-title {
+    height: auto !important;
+    margin-bottom: 25px;
+}
+
+.gs-visibleUrl {
+    margin-bottom: 6.4px !important;
+}
+
+.gsc-control-cse .gs-spelling,
+.gsc-control-cse .gs-result .gs-title,
+.gsc-control-cse .gs-result .gs-title * {
+    font-size: 22.4px !important;
+    font-family: 'Open Sans', Arial, Helvetica, sans-serif !important;
+    font-weight: 700 !important;
+    text-decoration: none !important;
+}
+
+/* Result Body Text */
+
+.gs-bidi-start-align.gs-snippet {
+    @extend p;
+    font-family: 'Open Sans', Arial, Helvetica, sans-serif !important;
+    font-size: 16px !important;
+    font-weight: 400;
+}
+
+.gs-bidi-start-align.gs-snippet b,
+.gs-visibleUrl b {
+    font-style: italic !important;
+    font-weight: 100 !important;
+}
+
+/* Paging Buttons */
+
+.gsc-cursor-page {
+    @extend .btn;
+}
+
+/* Results Info */
+
+.gsc-above-wrapper-area-container {
+	margin-bottom: 15px !important;
```

Comment:
update spacing as seen in [previous PR](https://github.com/baltimorecounty/BCPL-assets/pull/601#)

Reply:
ready for another look